fhir-model/src/main/java/com/ibm/fhir/model/parser/FHIRJsonParser.java

```diff
@@ -1283,10 +1283,14 @@ private Base64Binary parseBase64Binary(java.lang.String elementName, JsonValue j
                 checkForUnrecognizedElements(Element.class, jsonObject);
             }
             parseElement(builder, jsonObject);
+        } else if (_jsonValue != null) {
+            throw new IllegalArgumentException("Expected: OBJECT but found: " + _jsonValue.getValueType() + " for element: _" + elementName);
```

Comment:
a lot of ":" used in the message, maybe better to either separate with "," e.g, element: xxx, expected: xxxx, found: xxxx. or maybe use () or ' '.

Reply:
This is consistent with the other error messages for the parser. If I changed this, I'd have to change them all. Maybe a future endeavor.